pkg/config/configsections/config_section_test.go

```diff
@@ -47,7 +47,7 @@ const (
 	// deploymentName is the name of the deployment
 	deploymentName = "deployment-one"
 	// deploymentReplicas no of replicas
-	deploymentReplicas = "1"
+	deploymentReplicas = 1
```

Comment:
there's no function that tests parsing this. I think it's better to add some UT.

Reply:
Done.